source/sync/data-access-patterns/permissions.txt

```diff
@@ -489,3 +653,26 @@ A user with write permissions for a given collection can modify the value of any
 field of any object matching the client-side query. Write permissions require
 read permissions, so any user that can modify data can also view that data
 before and after it's modified.
+
+.. _flexible-sync-field-level-permissions:
+
+Field-Level Permissions
+^^^^^^^^^^^^^^^^^^^^^^^
+
+You can refine document-level permissions on a per-field basis with the
+``fields`` and ``additional_fields`` properties of the **Role** configuration
+object.
+
+If the user does not have document-level permissions to a document, field-level
+permissions cannot grant access to that document. Field-level permissions can
+only refine the permissions on documents that a user can access. For example,
+you can use field-level permissions to forbid writes on a specific field of a
+document that a user can otherwise write to.
+
+``additional_fields`` defines the field-level permissions for fields that you
```

Comment:
Hate to see a sentence start with lowercase. How about "The ``additional_fields``property defines..."